lib/vanagon/component/dsl.rb

```diff
@@ -316,8 +316,12 @@ def add_source(url, ref: nil, sum: nil)
       # @param mode [String] octal mode to apply to the directory
       # @param owner [String] owner of the directory
       # @param group [String] group of the directory
-      def directory(dir, mode: nil, owner: nil, group: nil)
-        @component.directories << Vanagon::Common::Pathname.new(dir, mode: mode, owner: owner, group: group)
+      def directory(dir, mode: nil, owner: nil, group: nil, wix_id: nil)
```

Comment:
No, https://github.com/puppetlabs/puppet_for_the_win/blob/master/wix/puppet.wxs#L151 is a good example of what `wix_id` could be set to